### DISCUSSION TITLE
I want to ask, Is didn't update for deno Doc?

### DISCUSSION THREAD

**Discussion by cuobiezi** [denoland/deno#15167] [Q&A] [answered]
I try to do follow [doc](https://deno.land/manual/contributing/building_from_source) , got some problems. 
1. build fail with "error: failed to run custom build command for `deno_ffi` v0.47.0".
2. I got a failed build directory target , And the subdirectories is "target/debug/build" . But the doc told me to run 
```shell
./target/debug/deno run cli/tests/testdata/002_hello.ts
```
Would help to explain them. Thanks

**Comment by kitsonk** [comment, marked as answer]
Have you updated submodules?
